test-cloud/test-cloud-ha-cep/pom.xml

```diff
@@ -123,6 +123,16 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
```

Comment:
Would it have sense to declare these plugins directly in test-cloud?

Reply:
Good point. I've moved these plugins to test-cloud module.